eox_nelp/signals/tests/test_tasks.py

```diff
@@ -355,24 +359,147 @@ def test_social_user_not_found(self):
 
 class CreateExternalCertificateTestCase(unittest.TestCase):
     """Test class for create_external_certificate function"""
+    def setUp(self):
```

Comment:
before your changes we had just the task create_external_certificate and the `CreateExternalCertificateTestCase` but now we have `create_external_certificate`, `create_external_certificate_directly` and `trigger_external_certificate_sqs` so every task requires its own test class, so please implement independent classes for every task instead of having one general test class